packages/uswds-core/src/styles/functions/units/px-to-user-em.scss

```diff
@@ -0,0 +1,24 @@
+@use "root";
+@use "./px-to-rem.scss" as *;
+@use "./rem-to-user-em.scss" as *;
+
+/*
+----------------------------------------
+px-to-user-em()
+----------------------------------------
+Converts a value in px to a value in em
+----------------------------------------
+*/
+
+@function px-to-user-em($pixels) {
+  @if unit($pixels) != "px" {
+    @error 'This value must be in pixels';
+  }
+  $px-to-user-em: px-to-rem($pixels);
+  $px-to-user-em: rem-to-user-em($px-to-user-em);
+
+  @return $px-to-user-em;
+}
```

Comment:
I opted not to re-invent the wheel and use functions we already had to convert pixels to rem.

Alternatively, we could use the following sass math conversion if we don't want to rely on the other functions:

```scss
  @if unit($pixels) != "px" {
    @error 'This value must be in pixels';
  }
  $px-to-user-em: (math.div($pixels, root.$root-font-size-equiv)) * 1em;
  $px-to-user-em: math.div(math.round($px-to-user-em * 100), 100);
```